### PR TITLE
Increase required oC version and own app version

### PR DESCRIPTION
**Pull request by nickvergessen**
@LukasReschke @DeepDiver1975 and another app

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
-		<author>Frank Karlitschek</author>
-		<shipped>true</shipped>
-		<default_enable/>
-		<require>4.9</require>
-		<types>
-				<filesystem/>
-		</types>
-		<ocsid>166038</ocsid>
+	<licence>AGPL</licence>
+	<author>Frank Karlitschek</author>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: please add @nickvergessen here

**Comment by DeepDiver1975** [comment]
:+1:

File: appinfo/info.xml (hunk 1 above)
Comment (nickvergessen) [inline appinfo/info.xml]: done

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**